### DISCUSSION TITLE
Ejemplo SeriesCapas.

### DISCUSSION THREAD

**Discussion by palacejohn** [AndresRamos/Contpaqi#101] [Q&A]
Hola, buen día!

Ayuda con ejemplo para agregar Series a un movimiento, varias series a un movimiento; es posible eso o debe agregarse una serie por movimiento?

**Comment by AndresRamos** [comment]
Ya se agrego un ejemplo de como generar un movimiento con series, ya esta disponible en `next-release` y estara en produccion en el proximo release.

Para crear un movimiento con series, crear un movimiento como normalmente lo haces y luego le agregas las series de tipo `SeriesCapas` a la propiedad `SeriesCapas` que es una colleccion.

```csharp
public int CrearMovimientoConSeries(int documentoId)
{
    var movimiento = new Movimiento
    {
        Producto = new Producto { Codigo = "PRODSERIES" },
        Almacen = new Almacen { Codigo = "PRUEBA" },
        Unidades = 1,
        Precio = 100,
        SeriesCapas = new List<SeriesCapas> { new() { Series = "123456" } }
    };

    return _movimientoService.Crear(documentoId, movimiento);
}
```
